gstd/src/exec/basic.rs

```diff
@@ -86,7 +76,7 @@ pub fn exit(inheritor_id: ActorId) -> ! {
 
 /// Resume previously paused message handling.
 ///
-/// Suppose a message has been paused using the [`wait`] function,
+/// Suppose a message has been paused using the [`gcore::exec::wait`] function,
```

Comment:
ref to gcore doc in gstd seems weird
```suggestion
/// Suppose a message has been paused using the [`crate::exec::wait`] function,
```